cmd/agent/proc.go

```diff
@@ -202,6 +206,10 @@ func (p *process) baseLogDir() string {
 	return path.Join(logDir, fmt.Sprintf("%s_%s", p.Type, p.ID))
 }
 
+func (p *process) storeDataDir(addr string) string {
+	return path.Join(dataDir, fmt.Sprintf("%s_%s", p.Type, addr))
```

Comment:
naming addr here is not proper, maybe, because some processes have no listening address. 

we can think another name.